application/src/main/java/run/halo/app/core/extension/attachment/endpoint/AttachmentEndpoint.java

```diff
@@ -102,6 +105,28 @@ public RouterFunction<ServerResponse> endpoint() {
                         ))
                     .response(responseBuilder().implementation(Attachment.class))
                     .build())
+            .POST("/attachments/external-transfer", contentType(MediaType.APPLICATION_JSON),
```

Comment:
```suggestion
            .POST("/attachments/-/upload-from-url", contentType(MediaType.APPLICATION_JSON),
```

Reply:
同时这里应该需要定义一个 rules 在附件相关的角色模板中，否则此 API 只能允许超级管理员调用

Reply:
不太理解这里添加一个 `-` 的作用是什么。直接改名为 `upload-from-url` 是否更恰当?

```suggestion
            .POST("/attachments/upload-from-url", contentType(MediaType.APPLICATION_JSON),
```

Reply:
因为我们的 URL 要符合 Resource 规范才能正确配置权限。这里 external-transfer，一是无法正常配置权限，二是占用了一个名称为 metadata.name 为 external-transfer 的附件。

Reply:
> 因为我们的 URL 要符合 Resource 规范才能正确配置权限。这里 external-transfer，一是无法正常配置权限，二是占用了一个名称为 metadata.name 为 external-transfer 的附件。

了解了。那我这边改一下。